docs/source/getting-started/index.rst

```diff
@@ -33,7 +33,7 @@ Make sure that you have the Daml SDK, Java 11 or higher, and Visual Studio Code
 
 You will also need some common software tools to build and interact with the template project:
 
-- `Node <https://nodejs.org/en/>`_ and the associated package manager ``npm``. Use the `Active LTS <https://nodejs.org/en/about/releases/>`_ Node version, currently ``v16`` (check with ``node --version``). If you have another version, see `this link <https://docs.npmjs.com/downloading-and-installing-node-js-and-npm>`_ for additional installation options. 
+- `Node <https://nodejs.org/en/>`_ and the associated package manager ``npm``. This guide has been tested against Node 18.
```

Comment:
This happens through manual testing, despite our automated tests running on Node 14.